app/qml/FeaturePanel.qml

```diff
@@ -175,8 +175,9 @@ Drawer {
                     property color backgroundColorInactive: "grey"
                     property color fontColor: InputStyle.fontColor
                     property color activeColor: InputStyle.fontColor
+                    property color attentionColor: "#aa0000"
                     property color normalColor: InputStyle.panelBackgroundLight
-                    property real cornerRadius: 1
+                    property real cornerRadius: 15 * QgsQuick.Utils.dp
```

Comment:
Umm.... I meant to turn 15 into `X * dp` while keeping the desired radius - `15*dp` is going to be too much :)

Reply:
I was thinking to decrease the value about the half, but actually now it seems fine to me. However, I am easy to have smaller radius.